### ISSUE TITLE
Load a few other Modern Perl values by default

### ISSUE THREAD

**Issue by srchulo** [iamalnewkirk/do#251]
I think it would be nice if

```
use Data::Object;
```

Loaded a few other modern perl values. For instance: 

- [Modern::Perl](https://metacpan.org/pod/Modern::Perl) sets the [mro](https://metacpan.org/pod/mro) to [C3](https://metacpan.org/pod/mro#The-C3-MRO). 
- `Modern::Perl` also loads [IO::File](https://metacpan.org/pod/IO::File)
- `Modern::Perl` also loads [IO::Handle](https://metacpan.org/pod/IO::Handle)
- [Mojo::Base](https://metacpan.org/pod/Mojo::Base#DESCRIPTION) loads [utf8](https://metacpan.org/pod/utf8)

I think these would be good additions!

**Comment by iamalnewkirk** [comment]
Good point! Thanks for the feedback. I'll take a look at the startup/performance costs of adding those, although I'm not really in the performance optimization phase yet. They would make good additions.
